lesson03.find-a-pair/script/find-a-pair.js

```diff
@@ -0,0 +1,86 @@
+import { Game } from './game.js';
+
+const game = new Game();
+
+const audio_game_over = new Audio('../audio/game-over.mp3');
```

Comment:
It doesn;'t worng because of wrong path. since everything is included in `index.html`, it should be with a single dot:
```
./audio/game-over.mp3
```